audit/org_kubernetes.io/iam.json

```diff
@@ -66,6 +66,12 @@
       ],
       "role": "roles/iam.securityReviewer"
     },
+    {
+      "members": [
+        "group:k8s-infra-gcp-org-admins@kubernetes.io"
+      ],
+      "role": "roles/orgpolicy.policyAdmin"
```

Comment:
Expected, part of https://github.com/kubernetes/k8s.io/pull/2010 being deployed (https://github.com/kubernetes/k8s.io/pull/2010/commits/ebece4e7c78136ea161d135c4bce66952d67db70)